apps/api/src/hyp3_api/api-spec/openapi-spec.yml

```diff
@@ -161,6 +188,13 @@ components:
         job_specification:
           $ref: "#components/schemas/subscription_job_specification"
 
+    subscriptions_update_expression:
+      description: Change/update a subscription
+      type: object
+      properties:
```

Comment:
should include `additionalProperties: false`, and I lean towards making `end` required